d3rlpy/constants.py

```diff
@@ -47,3 +47,8 @@ class ActionSpace(Enum):
 class PositionEncodingType(Enum):
     SIMPLE = "simple"
     GLOBAL = "global"
+
+
+class LoggingStrategyEnum(Enum):
```

Comment:
Could you rename this to `LoggingStrategy`?

Reply:
Of course. I just changed it.